crates/uv/src/commands/reporters.rs

```diff
@@ -78,6 +83,20 @@ impl uv_installer::DownloadReporter for DownloadReporter {
 
     fn on_complete(&self) {
         self.progress.finish_and_clear();
+        if let Some(length) = self.progress.length() {
+            let s = if length == 1 { "" } else { "s" };
+            writeln!(
+                self.printer.stderr(),
+                "{}",
+                format!(
+                    "Downloaded {} in {}",
+                    format!("{} package{}", length, s).bold(),
+                    elapsed(self.start_time.elapsed())
+                )
+                .dimmed()
+            )
+            .unwrap();
```

Comment:
How does the progress bar implementation get around this? Do they just eat errors?